Questions/ReverseSubArray.ts

```diff
@@ -0,0 +1,22 @@
+function reverseSubArray(arr:number[],k:number):void{
+    let n=arr.length
+    for(let i=0;i<k;i=i+k){
+        let start=i;
+        let end=Math.min(i+k-1,n-1);
+        let temp:number;
+
+        while(start<end){
+            temp=arr[start];
+            arr[start]=arr[end];
+            arr[end]=temp;
+            start++;
+            end--;
+        }
+    }
+}
+
+const arr=[1,4,7,3,5,3];
+
+reverseSubArray(arr,3);
```

Comment:
not working after first iteration it converts first set only.
example run with => reverseSubArray(arr,2);

got this [4, 1, 7, 3, 5, 3]